### DISCUSSION TITLE
Previous versions of the documentation

### DISCUSSION THREAD

**Discussion by JackDI1** [NatLabRockies/floris#929] [Q&A] [answered]
Hi,

I was wondering if there is a way of getting a copy of the documentation for previous versions of Floris. I was in the middle of implementing Floris as part of a project and took a break. In that time, Floris V4 was released, and the documentation was updated! Is there any way I can find the V3 documentation so I can complete the implementation before potentially upgrading to V4?

Thanks,
Jack

**Comment by misi9170** [comment, marked as answer]
Hi @JackDI1 ,

You can find v3 documentation in the [docs/](https://github.com/NREL/floris/tree/v3/docs) directory of the [v3 FLORIS branch](https://github.com/NREL/floris/tree/v3). This is the source code for the v3 documentation, but is mostly in markdown so is fairly easy to read as is.

If you'd like to build the html for that and use it as though it was a website, you can follow [these instructions](https://nrel.github.io/floris/dev_guide.html#documentation) to build to html and then open it in your browser.

Cheers,
Misha
